### PR TITLE
login-signup

### PR DESCRIPTION
**Pull request by tuanhnt1712**
signup

### CODE REVIEW

**Comment by tuanhnt1712** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -23,28 +23,23 @@
...
             </ul>
           </li>
         <% else %>
           <li>
-            <%= link_to new_user_registration_path do %>
-              <span class="glyphicon glyphicon-user"></span>
-              <%= t ".signup" %>
-            <% end %>
+            <%= link_to t(".signup"), "#signup", "data-toggle" => "modal",
+              "data-target" => "#signup", :class => "btn btn-small" %>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: `class: "btn btn-small"`

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -23,28 +23,23 @@
...
           </li>
           <li>
-            <%= link_to new_user_session_path do %>
-              <span class="glyphicon glyphicon-log-in"></span>
-              <%= t ".login" %>
-            <% end %>
-          </li>
+            <%= link_to t(".login"), "#login", "data-toggle" => "modal",
+              "data-target" => "#login", :class => "btn btn-small" %>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: `class: "btn btn-small"`

File: app/assets/stylesheets/custom.scss [hunk 3]
```diff
@@ -15,3 +17,22 @@
...
+h1 {
+  font-size: 2em;
+  letter-spacing: -2px;
+  line-height: 1;
+  margin-bottom: 30px;
+  text-align: center;
+}
+
+.navbar-custom .navbar-brand {
+    color: $blue;
```
Comment (dieunb) [inline app/assets/stylesheets/custom.scss]: indent

File: app/assets/stylesheets/custom.scss [hunk 4]
```diff
@@ -15,3 +17,22 @@
...
+  }
```
Comment (dieunb) [inline app/assets/stylesheets/custom.scss]: indent

File: app/assets/stylesheets/custom.scss [hunk 5]
```diff
@@ -15,3 +17,22 @@
...
+.navbar-custom .nav li a {
+    color: $blue;
```
Comment (dieunb) [inline app/assets/stylesheets/custom.scss]: indent

File: app/assets/stylesheets/custom.scss [hunk 6]
```diff
@@ -15,3 +17,22 @@
...
+  }
```
Comment (dieunb) [inline app/assets/stylesheets/custom.scss]: indent

**Comment by dieunb** [comment]
commented

**Comment by tuanhnt1712** [comment]
ready

**Comment by dieunb** [comment]
merged

**Pull request merged**